databook/glossary.md

```diff
@@ -113,6 +139,19 @@ Neurogliaform cell
 Receptive field
   definition here
 
+regular spiking unit
+RS
+  Neurons that, when injected with a long step of current in the context of
+  intracellular recordings, show spike frequency adaptation where the rate of
+  spiking decreases over time. These neurons also have longer (or wider) action
+  potentials, and lower spike rates even when injected with large currents due
+  to hyperpolarization after each action potential. These are the most common
+  type of neurons in the mammalian cortex, and are often associated excitatory
+  neurons. In extracellular recordings, neurons with longer action potentials
+  are also sometimes referred to as regular spiking neurons, a feature which is
+  used to putatively classify them as excitatory neurons
```

Comment:
VIP as also RS.

Reply:
Added text to emphasize that it's not exclusive